conf.py

```diff
@@ -240,16 +247,16 @@
 #     )
 
 POSTS = (
-    ("posts/*.rst", "blog", "post.tmpl"),
     ("posts/*.md", "blog", "post.tmpl"),
+    ("posts/*.rst", "blog", "post.tmpl"),
     ("posts/*.txt", "blog", "post.tmpl"),
     ("posts/*.html", "blog", "post.tmpl"),
 )
 PAGES = (
-    ("pages/*.rst", "pages", "page.tmpl"),
-    ("pages/*.md", "pages", "page.tmpl"),
-    ("pages/*.txt", "pages", "page.tmpl"),
-    ("pages/*.html", "pages", "page.tmpl"),
+    ("pages/*.md", "", "page.tmpl"),
+    ("pages/*.rst", "", "page.tmpl"),
+    ("pages/*.txt", "", "page.tmpl"),
+    ("pages/*.html", "", "page.tmpl"),
```

Comment:
Does `("pages/index.md", "", "homepage.tmpl"),` make sense? That means only the index page builds the homepage bands and we can leave `("pages/*.md", "", "page.tmpl")` for something more generic. Or should we do that in a later PR?

Reply:
So the "" comes from noticing that the index page had rendered to pages/index.html, and obviously we want the frongpage at the root, and I blatantly copied the conf.py from Nikola's own site to fix it. 

It doesn't matter what's in that conf entry for the bands, as that conf option just controls the path things get rendered to. Pages all use `page.tmpl` as a base, and that's where the bands are currently. I think it's fine for now, but longer term we'll either want to
- hardcode the frontpage (as Nikola does in https://github.com/getnikola/nikola-site/blob/master/stories/index.html) but then we have to put the bands code in the single file
- Use something like https://getnikola.com/theming.html#identifying-and-customizing-different-kinds-of-pages-with-a-shared-template to detect the frongpage in page.tmpl and only load the bands then

I favour the second but we don't have to fix it until we have more than one page :)